packages/ui-top-nav-bar/src/TopNavBar/TopNavBarLayout/SmallViewportLayout/props.ts

```diff
@@ -113,6 +113,11 @@ type SmallViewportLayoutOwnProps = {
    * Callback fired when an item is selected in the dropdown menu
    */
   onDropdownMenuSelect?: DrilldownProps['onSelect']
+
+  /**
+   * Stay away...
+   */
+  renderNavbarStartDangerousHack?: React.ReactNode
```

Comment:
Maybe expand the type with HTMLElement if someone wants to add a simple html tag, otherwise nice job.

Reply:
regular html tags should be compatible with `ReactNode`, at least in my testings it didn't throw any error for them